### PR TITLE
ES2015ify

### PR DESCRIPTION
**Pull request by SamVerschueren**
Did 2 commits. The first one cleans up some stuff, the second one es2015ifies the module. You/I can squash it if you like.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
-	items.forEach(function (item) {
-		devices.filter(function (device) {
-			return device.name.split(' ').join('').indexOf(item) !== -1;
-		}).forEach(function (device) {
-			ret.push(device);
-		});
+	items.forEach(item => {
+		devices
+			.filter(device => device.name.split(' ').join('').indexOf(item) !== -1)
+			.forEach(device => {
```
Comment (SamVerschueren) [inline index.js]: Odd, this is indented correctly locally :s.

**Comment by SamVerschueren** [comment]
// @sindresorhus

File: index.js [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
 
-	items.forEach(function (item) {
-		devices.filter(function (device) {
-			return device.name.split(' ').join('').indexOf(item) !== -1;
-		}).forEach(function (device) {
-			ret.push(device);
-		});
+	items.forEach(item => {
+		devices
+			.filter(device => device.name.split(' ').join('').indexOf(item) !== -1)
```
Comment (sindresorhus) [inline index.js]: Use `.contains()`

File: index.js [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
-	var ret = [];
+	const ret = [];
 
-	items.forEach(function (item) {
-		devices.filter(function (device) {
-			return device.name.split(' ').join('').indexOf(item) !== -1;
-		}).forEach(function (device) {
-			ret.push(device);
-		});
+	items.forEach(item => {
```
Comment (sindresorhus) [inline index.js]: for-of loop

File: index.js (hunk 2 above)
Comment (SamVerschueren) [inline index.js]: You mean [includes](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/String/includes)?

File: index.js [hunk 4]
```diff
@@ -1,27 +1,25 @@
...
-		}).forEach(function (device) {
+	for (const item of items) {
+		const filteredDevices = devices.filter(device => device.name.split(' ').join('').includes(item));
+
+		for (const device of filteredDevices) {
 			ret.push(device);
-		});
-	});
+		}
+	}
```
Comment (SamVerschueren) [inline index.js]: Is this block ok?

File: index.js (hunk 2 above)
Comment (sindresorhus) [inline index.js]: Yes

File: index.js (hunk 4 above)
Comment (sindresorhus) [inline index.js]: Yup

Comment (SamVerschueren) [inline index.js]: Another option would be `ret = ret.concat(devices.filter(...))`

File: index.js [hunk 5]
```diff
@@ -1,27 +1,25 @@
 'use strict';
-var arrify = require('arrify');
-var devices = require('./data.json').devices;
+const arrify = require('arrify');
+const devices = require('./data.json').devices;
 
-module.exports = function (items) {
+module.exports = items => {
 	items = arrify(items);
```
Comment (sindresorhus) [inline index.js]: Not sure why we arrify it here. The docs only documents an array. I think we can drop this and rather do an array type check.

File: readme.md [hunk 6]
```diff
@@ -32,7 +32,7 @@ Returns a promise for an array of viewports.
 
 #### items
 
-Type: `array`  
+Type: `array`<br>
```
Comment (sindresorhus) [inline readme.md]: Array

File: index.js (hunk 4 above)
Comment (sindresorhus) [inline index.js]: The "Another option" would indeed be better.

**Comment by SamVerschueren** [comment]
Fixed and updated the docs because it's synchronous now.

File: index.js [hunk 7]
```diff
@@ -1,27 +1,22 @@
 'use strict';
-var arrify = require('arrify');
-var devices = require('./data.json').devices;
+const devices = require('./data.json').devices;
 
-module.exports = function (items) {
-	items = arrify(items);
+module.exports = items => {
+	if (items && !Array.isArray(items)) {
+		throw new Error(`Expected an \`Array\`, found a \`${typeof items}\``);
```
Comment (sindresorhus) [inline index.js]: found => got

Comment (SamVerschueren) [inline index.js]: 1. Expected `Array`, got `string`
2. Expected an `Array`, got a `string`
3. Expected an `Array`, got `string`

?

Comment (sindresorhus) [inline index.js]: I prefer `1`. It's right to the point.

**Comment by SamVerschueren** [comment]
Sorry, on the train so can't squash right now :p

**Comment by sindresorhus** [comment]
Goodie. Thanks Sam.

@kevva Can you do a major release?

**Comment by SamVerschueren** [comment]
ping @kevva for new release :)

**Comment by kevva** [comment]
Sorry guys for dropping the ball on some stuff over the last weeks. Work got in the way. However, thanks for this, great as always :).

**Pull request merged**